subprojects/scala/src/main/java/org/gradle/api/tasks/scala/ScalaDoc.java

```diff
@@ -133,6 +136,28 @@ public void setTitle(@Nullable String title) {
         this.title = title;
     }
 
+    /**
+     * Returns the amount of memory allocated to this task.
+     *
+     * @since 6.5
+     */
+    @Internal
+    @Nullable
+    public Property<String> getMaxMemory() {
+        return maxMemory;
+    }
+
+    /**
+     * Sets the amount of memory allocated to this task.
+     *
+     * @param maxMemory The amount of memory
+     *
+     * @since 6.5
+     */
+    public void setMaxMemory(String maxMemory) {
```

Comment:
This setter needs to be removed.  The Property should be set through `getMaxMemory`

Reply:
oh yeah that makes total sense, removed in https://github.com/gradle/gradle/pull/12718/commits/aeddf3e3e20819ce6ab6bb8d21d50466bb068095